pkg/operator/controller/status.go

```diff
@@ -232,3 +209,35 @@ func setStatusCondition(oldConditions []configv1.ClusterOperatorStatusCondition,
 
 	return newConditions
 }
+
+// statusesEqual compares to ClusterOperatorStatus values.  Returns true the
```

Comment:
compares *two* ClusterOperatorStatus... Returns true *if* the...

Reply:
Whoops! Fixed.  Thanks!